plugin.json

```diff
@@ -1,10 +1,10 @@
 {
     "id": "tesseract-ocr",
-    "version": "1.0.2",
+    "version": "1.1.0",
```

Comment:
IMO could even be a v2, since we are renaming the plugin and adding a new lib

Reply:
ok let's go for v2.0.0 !